app/uk/gov/hmrc/organisationsmatchingapi/connectors/OrganisationsMatchingConnector.scala

```diff
@@ -35,12 +35,22 @@ class OrganisationsMatchingConnector @Inject()(servicesConfig: ServicesConfig,
 
   private val baseUrl = servicesConfig.baseUrl("organisations-matching")
 
+  def headerConvert(hc: HeaderCarrier): Seq[(String, String)] = {
+    def getValue(headerName: String) = hc.headers(Seq(headerName)).head._2
+    
+    val appId = getValue("X-Application-ID")
```

Comment:
Using this string constant twice - I'd define them in a list (requiredHeaders or some such), and have getValue return the tuple of (headerName, headerValue) - headerConvert would then just map the sequence of required Headers by calling getValue and return.